src/libs/actions/BankAccounts.js

```diff
@@ -402,6 +406,7 @@ function fetchFreePlanVerifiedBankAccount(stepToOpen) {
                     achData.isInSetup = !bankAccount || bankAccount.isInSetup();
                     achData.bankAccountInReview = bankAccount && bankAccount.isVerifying();
                     achData.domainLimit = 0;
+                    achData.state = lodashGet(achData, 'state', '');
```

Comment:
Why did we need to add this here?

Reply:
It adds a default state to `achData` so we override the temporary state when merging [here](https://github.com/Expensify/App/blob/22c872cfb2409cc0e1e2b774102f2da4a0db09f6/src/libs/actions/BankAccounts.js#L70)

Reply:
Does the value not already exist in `bankAccount.toACHData()`?

Reply:
Updated!